package.json

```diff
@@ -8,8 +8,8 @@
   },
   "scripts": {
     "build": "rimraf lib && tsc",
-    "lint": "tslint \"src/**/*.ts\" --type-check --project ./tsconfig.json",
-    "lint:fix": "tslint --fix \"src/**/*.ts\" --type-check --project ./tsconfig.json",
+    "lint": "tslint \"src/**/*.ts\" --project ./tsconfig.json",
```

Comment:
Hum... why did you remove the type-check?

Reply:
Probably this can be safely reverted